tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -841,12 +845,12 @@ in that order.
         _state.exit()
         self._state = target_state
         _state.enter(msg)
-        Events.emit_signal("player_state_changed", _state.name)
+        Events.player_state_changed.emit(_state.name)
 
 
     func _on_state_changed(previous, new):
         print("state changed")
-        emit_signal("state_changed")
+        state_changed.emit()
```

Comment:
I've only changed the syntax here, but functionality-wise this seems strange: the function that's connected to the signal re-emits the signal. I'd think the intent was to emit it in the setter instead.